### ISSUE TITLE
Sound Flags Accessibility

### ISSUE THREAD

**Issue by zacjansheski** [musescore/MuseScore#21739]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Bug description

Sound Flags are only accessible by clicking the Sound Flags button next to staff text.
Needs to be accessible via alt+arrow

Also the screen reader doesn't read the sound flags options.
Rather than reading "Snares On" and "Snares Off" it just says "Button"

### Steps to reproduce

Try to use sound flags with keyboard nav and screen reader

### Screenshots/Screen recordings

(Apologies for my awkward keyboard nav skills)
https://github.com/musescore/MuseScore/assets/69917893/844fb270-f7aa-4ff1-a415-859134822d0f

### MuseScore Version

4.3

### Regression

No.

### Operating system

Mac OS 14
